### DISCUSSION TITLE
PP-OCR V3 检测模型微调不起作用

### DISCUSSION THREAD

**Discussion by Alanhzl** [PaddlePaddle/PaddleOCR#12704] [Q&A]
使用V3检测模型ch_PP-OCRv3_det去检测文本区域时，有的图检测不出文本框。

用V3训练模型调用下面的命令：
python3 tools/infer_det.py -c configsChange/det/ch_PP-OCRv3_det_cml.yml -o Global.infer_img="./train_data/det/testData/origin-13000-13999/" Global.pretrained_model="./pretrain_models/V3/ch_PP-OCRv3_det_distill_train/best_accuracy"

用上面的命令检测应用场景的数据集，然后将检测不出来的图挑出来，使用PPOCRLabel手动标注文本区域。由于检测不出来的图大都文字有些弯曲，所以使用了多点标注。

用挑出来的图进行V3检测模型的训练，训练后的模型基本和原来的没什么差别。看output中的文件修改时间发现，best_accuracy大约在40轮后就没有再变过了，后面的400+轮训练似乎都是无效的。

样例图
![000013546](https://github.com/PaddlePaddle/PaddleOCR/assets/39329022/a8c788e5-1149-4913-b5e3-90adc8e907a8)
![000013548](https://github.com/PaddlePaddle/PaddleOCR/assets/39329022/6641ae2b-dc7d-4345-9578-a5d75aef2738)

标注图
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/39329022/75e1b744-db29-44be-ae09-a5d63f75d951)
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/39329022/a566113c-1087-48d1-9bdb-cf4a7459990a)

训练时输出的文件
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/39329022/677e220e-c361-4a8f-820f-f36a458d3bb7)

请问对于这些识别不了的图，微调检测模型时要怎么做？上面的做法还有什么遗漏的地方？
是不是在微调时，多点标注的数据是需要改代码才能训练的？

**Comment by GreatV** [comment]
建议参考 https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_ch/finetune.md

**Reply by Alanhzl** [reply]
> 建议参考 https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_ch/finetune.md

文本检测模型微调部分没有说明这种弯曲文字的处理。
我按照微调的说明，将检测不出来的图筛选后，大约数量1000+的数据集用PPOCRLabel多点标注，用这份数据对ch_PP-OCRv3_det的训练模型进行微调训练，训练后的模型效果和训练前基本没差，还是无法检测上面的弯曲文字。
多点标注的数据，是不是需要修改代码后再进行训练？

**Comment by Alanhzl** [comment]
各位大佬们，这种弯折文字具体要怎么训练才能检测出来？我看#1966这里提到：
检测： 标注的时候使用多点标注，修改网络的后处理部分，添加多边形输出

这个修改网络后处理部分，添加多边形输出是要修改哪里？
